### ISSUE TITLE
conda-installed openorb version

### ISSUE THREAD

**Issue by ysBach** [oorb/oorb#178]
If I install openorb by conda/mamba, the version is ``1.0.1.2019102``,
which is different from the released version (1.3).

**Comment by rhiannonlynne** [comment]
This is true!  We've been struggling a bit with the conda packaging .. the people who set that up initially have been busy and not had time to update it for recent changes in oorb but the updates in oorb are such that a simple update in conda-forge doesn't work. 

You could try, as a workaround, installing from pypi. 
`pip install pyoorb` will get you the 1.3 version openorb installation, although without the associated data. You could then install the de405/whichever ephemeris by hand (or see if the openorb-data-xxx will give you the data you need). 
It's not always great to mix a pip and conda installation in the same environment, so this approach carries some risk. 

We're working on the conda packaging and will hopefully be able to upgrade soon.
